packages/react-components/react-utilities/src/compose/overrideSlotClasses.ts

```diff
@@ -0,0 +1,9 @@
+import { mergeClasses } from '@griffel/react';
+import { SLOT_USER_CLASS_NAME_SYMBOL } from '../compose/constants';
+import { UnknownSlotProps } from '../compose/types';
+import { isSlot } from './isSlot';
+
+export const overrideSlotClasses = (slot: UnknownSlotProps, ...classNames: (string | false | undefined)[]) => {
```

Comment:
I wonder if it'd be possible to make the behaviour more self-explanatory with the name. e.g. `overrideSlotBaseClasses` to show that it's overriding the built-in styles, but _not_ overriding the user provided styles.

Reply:
maybe `overrideDefaultSlotClasses`?

Reply:
@behowell - Where will the existing fluent components call this? Maybe the hook that gives back the custom style hook could wrap the custom style hook call and call this afterwards if we wanted to make it automatic and not have to touch every component.

Reply:
The intent is for the custom style hook itself to call this function, as a replacement for `mergeClasses`. That way we aren't changing the behavior of existing hooks.

I looked briefly into a way to do this automatically so that style hooks don't need to be modified (perhaps aside from a single opt-in flag if we're worried about a breaking change), but it ended up being a bit hacky. I'll look into it a little more to see if there's another way...